stock-tracker/src/features/chart/component/chart.tsx

```diff
@@ -22,6 +22,25 @@ import { ChartTime } from "../redux/actions";
 
 const chartRanges: ChartTime[] = ["1D", "5D", "1M", "1Y", "5Y", "MAX"];
 
+const formatDateXAxis = (tickItem: string, current: ChartTime) => {
+  switch (current) {
+    case "1D":
+      return tickItem;
+    case "5D":
+      return moment(tickItem).format("dddd");
+    case "1M":
+      return moment(tickItem).format("MMM Do");
+    case "1Y":
+      return moment(tickItem).format("MMM Do");
+    case "5Y":
```

Comment:
`Case 1M` and `Case 1Y` can combine together

Reply:
Good point, committed that now.